### DISCUSSION TITLE
When Docker Hub will have change username feature?

### DISCUSSION THREAD

**Discussion by jinzzy-17** [docker/roadmap#456] [Q&A] [answered]
I want change to my username on Docker hub

**Comment by Xaypanya** [comment, marked as answer]
Unfortunately, you will need to create a new account

**Comment by codenamehaven** [comment]
> Unfortunately, you will need to create a new account

As my pair mentioned, alas, this is the only solution available at the moment. Thankfully, the status of this feature has been updated to be currently in consideration in the roadmap as of late. To stay updated and be notified, consider subscribing to the issue https://github.com/docker/roadmap/issues/44.
